src/snowflake/cli/plugins/nativeapp/artifacts.py

```diff
@@ -228,7 +235,9 @@ def build_bundle(
 
             # copy all files as children of the given destination path
             for source_path in source_paths:
-                symlink_or_copy(source_path, dest_path / source_path.name)
+                dest_child_path = dest_path / source_path.name
+                symlink_or_copy(source_path, dest_child_path)
+                mapped_files[source_path.resolve()] = dest_child_path
```

Comment:
Might be related to the discussion above, but why:
1. Is `source_path` being resolved?
2. Is only `source_path` being resolved but not `dest_child_path`?

Also, can you clarify the following:
If the user specified a glob pattern, then `source_path` will be the directories or files that qualify the source pattern. In case of directories, even though their contents are symlinked as well, `mapped_files` will only contain a kv pair of the directory to the dest, and not its contents.

Reply:
1. Both src/dest parts in `ArtifactDeploymentMap` are resolved, mostly to canonicalize windows paths `(C:\Users\RUNNER~1\` -> `C:\Users\runneradmin\`)
2. `dest_child_path` is also resolved (`dest_path` is resolved a few lines above)

Yes, `mapped_files` won't contain the contents of symlinked directories. It's only the files that we actually copied to the deploy directory.
You can see the logic in `project_path_to_deploy_path()` for cases where we look for a file under a symlinked directory.